### PR TITLE
adding alasaidi.md file

### PR DESCRIPTION
**Pull request by alasaidi**
adding my Bio plus update the spell check

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -104,7 +104,12 @@
     "Amira",
     "amirabayoumi",
     "Ashraf",
-    "Ashrafpoless"
+    "Ashrafpoless",
```
Comment (denisrossi) [inline .cspell.json:107]: There will be competing changes with @jgchoti [PR](https://github.com/HYF-Class22/home/pull/57/files). That's normal, be mindful of what happen next while you cooperate with your classmate.

The first that will successfully merge their PR will make the other resolve the conflicts. You can learn more [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line)

Comment (alasaidi) [reply .cspell.json:107]: @denisrossi I didn't understand exactly what you mean. Could you please provide more explanation?

Comment (denisrossi) [reply .cspell.json:108]: When more than one person modify the same file, that will generate what in git is called a conflict, you can find more informations [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line)
> Merge conflicts occur when competing changes are made to the same line of a file, or when one person edits a file and another person deletes the same file

Comment (alasaidi) [reply .cspell.json]: @denisrossi i think i solved

**Review by denisrossi — APPROVED** [review]

**Pull request merged**